packages/create-gatsby/src/__tests__/tracking.ts

```diff
@@ -0,0 +1,61 @@
+let isTrackingEnabled: () => boolean
+
+const get = jest.fn()
+const set = jest.fn()
+
+jest.doMock(`../get-config-store`, () => {
+  return {
+    getConfigStore: (): unknown => {
+      return {
+        get,
+        set,
+      }
+    },
+  }
+})
+
+describe(`isTrackingEnabled`, () => {
+  beforeEach(() => {
+    jest.resetModules()
+    isTrackingEnabled = require(`../tracking`).isTrackingEnabled
+  })
+
+  it(`is enabled by default`, async () => {
+    const enabled = isTrackingEnabled()
+    expect(enabled).toBeTrue()
+  })
+
+  it(`respects the setting of the config store`, async () => {
+    get.mockImplementationOnce(key => {
+      if (key === `telemetry.enabled`) {
+        return false
+      } else {
+        return true
+      }
+    })
+
+    const enabled = isTrackingEnabled()
+    expect(enabled).toBeFalse()
+
+    const cachedEnabled = isTrackingEnabled()
+    expect(cachedEnabled).toBeFalse()
+  })
+
+  describe(`isTrackingEnabled / process.env.GATSBY_TELEMETRY_DISABLED`, () => {
```

Comment:
```suggestion
  describe(`process.env.GATSBY_TELEMETRY_DISABLED`, () => {
```

It'll already be nested in Jest and have `isTrackingEnabled` as "parent"